src/odemis/model/_components.py

```diff
@@ -107,12 +107,14 @@ class Component(ComponentBase):
     Component to be shared remotely
     '''
 
-    def __init__(self, name, parent=None, children=None, daemon=None):
+    def __init__(self, name, parent=None, children=None, dependencies=None, daemon=None):
         """
         name (string): unique name used to identify the component
         parent (Component): the parent of this component, that will be in .parent
         children (dict str -> Component): the children of this component, that will
-            be in .children. Objects not instance of Component are skipped
+            be in .children. Objects not instance of Component are skipped.
+        dependencies (dict str -> Component): the dependencies of this component, that will
+            be in .dependencies. Objects not instance of Component are skipped
```

Comment:
Please drop this "Objects not instance of Component are skipped", as it's not correct.

Reply:
Still there?

Reply:
Oh, actually, you've removed it... but the wrong one. Please, put back the text for children, and remove for dependencies.

Reply:
Indeed, I didn't pay attention... Should be fixed now.